default/python/AI/freeorion_tools/_freeorion_tools.py

```diff
@@ -349,3 +351,31 @@ def get_ship_part(part_name: str):
         warning("Could not find part %s" % part_name)
 
     return part_type
+
+
+def named_int(name: str) -> int:
```

Comment:
Maybe `get_int_reference`?  It would be nice to include the verb to the function name.

Reply:
In FOCS they are called NamedReal and NamedInt, maybe get_named_int?

My long term idea is to mostly use them to initialise global variables in AIDependencies.py. So far the AI is started too early, however, I simply get None when I use them at global level.

Reply:
`get_named_int`  looks good.